addons/web/static/src/search/filter_menu/filter_menu.js

```diff
@@ -35,5 +36,5 @@ export class FilterMenu extends Component {
     }
 }
 
-FilterMenu.components = { CustomFilterItem };
+FilterMenu.components = { CustomFilterItem, DropdownItem: SearchDropdownItem };
```

Comment:
Apparently, this is not working. Selectable menuitems are rendered with the role menuitem instead of menuitemcheckbox and the attribute `aria-checked` is not present. Take a look to how this selected filter is being rendered:

`<span role="menuitem" tabindex="0" class="dropdown-item o_menu_item selected focus">Individuals</span>`

Reply:
Off course you're right.

I forgot that there are two implementations of the search menus in odoo 15.0.

Just force pushed the branch with the missing parts of the fix.

Could you check it again ?

Thanks.